lib/logstash/inputs/wmi.rb

```diff
@@ -47,23 +47,28 @@ def run(queue)
     @wmi = WIN32OLE.connect("winmgmts://")
     
     begin
-      @logger.debug("Executing WMI query '#{@query}'")
-      loop do
+      while !stop?
+        @logger.debug("Executing WMI query '#{@query}'")
```

Comment:
this should use structured logs, not string interpolation:

`
@logger.debug("Executing WMI query", :query => @query)
`